.circleci/config.yml

```diff
@@ -28,18 +18,6 @@ jobs:
     executor: validator
     steps:
       - checkout
-      - hmpps/install_aws_cli
```

Comment:
https://discuss.circleci.com/t/linux-machine-executor-images-2021-january-q1-update/38976

The machine we are using  ubuntu-2004:202101-01 has AWS CLI v2 pre-installed